packages/slice-machine/components/FormFields/utils/getFieldStyles.tsx

```diff
@@ -0,0 +1,46 @@
+interface getFieldStylesProps {
+  isError: boolean;
+  isDisabled: boolean;
+  isWarning: boolean;
+}
+
+export const getFieldStyles = ({
+  isError,
+  isDisabled,
+  isWarning,
+}: getFieldStylesProps) => {
+  const defaultFieldStyles = {
```

Comment:
That could also have been made using CSS & Classnames
```suggestion
  import cls from 'classnames'
  
  <span
    classname={cls(
      isError && 'error',
      isWarning && 'warning',
      isDisabled && 'disabled'
    )}
  ></span>
```

Reply:
that way the CSS handles the style, we usually avoid css in the TS, it usually is a last ressort for dynamic properties for instance.

Reply:
good idea